### ISSUE TITLE
report in braille how many items are in a list

### ISSUE THREAD

**Issue by gregjozk** [nvaccess/nvda#7455]
as @dkager advised at issue #7450 (comment) I'm opening a separate ticket for this.

### Expected behavior:
On webpages and other documents, there are also lists and tables with x items. I suggest that NVDA reports in braille how many elements are on a list as it does with speech. e.g.:
- speech: list with x items or table with x rows and y columns.
- braile: lst with x items or tb with x rows and y columns 

I don't expect that NVDA would tell me where I was positioned (8 of 15).

### Actual behavior:
nothing on braille display only speech
when list or table starts, the first element is marked with lst or tb abbreviation and that's it.

### System configuration:
NVDA version:
2017.2
NVDA Installed or portable:
both

Windows version:
w10 1703 with latest updates

**Comment by dkager** [comment]
> I don't expect that NVDA would tell me where I was positioned (8 of 15).

I agree this is the appropriate behavior for unordered lists. For ordered lists and tables the current index of an item is intrinsic, and reporting the total number of items is the topic of this issue.

**Comment by gregjozk** [comment]
Hi,

> I agree this is the appropriate behavior for unordered lists. For ordered lists and tables the current index of an item is intrinsic, and reporting the total number of items is the topic of this issue.

but speech feedback does not give this information either, so I don't expect this, however I will be glad, if this would be done.

**Comment by dkager** [comment]
Yep yep, I was agreeing with you. :)

**Comment by Adriani90** [comment]
I do not have a braille display to test, but I this has been implemented meanwhile. But I might be wrong though.

**Comment by ruifontes** [comment]
No, not yet...
In Braille it only shows lst...

**Comment by Andre9642** [comment]
I'm working on this issue through Braille Extender add-on. See https://github.com/Andre9642/BrailleExtender/pull/64

Currently, the output in braille is: `lst(n)`. E.g.: `lst(42)`.
This output will be soon customizable.
Can be tested with the development version of the add-on. Any suggestion are welcome!

**Comment by nvdaes** [comment]
I'm working on this locally. I'll create a PR.
